UPGRADE_GUIDE.md

```diff
@@ -138,7 +138,7 @@ active when the index route is active.
 </Route>
 
 // will be active only when home is active, not when about is active
-<IndexLink to="/">Home</Link>
+<Link to="/">Home</Link>
```

Comment:
This change is not accurate. You need to use an `IndexLink` to link directly to an `IndexRoute`.